Source/SelfService/Web/theme/dropDownMenu.tsx

```diff
@@ -19,56 +19,46 @@ type DropDownMenuItem = {
     displayValue: string;
 };
 
-const useStyles = makeStyles((theme: Theme) =>
-    createStyles({
-        formControl: {
-            'margin': theme.spacing(1),
-            'minWidth': 220,
-            '& .MuiOutlinedInput-input': {
-                color: 'white'
-            },
-            '& .MuiInputLabel-root': {
-                color: 'white'
-            },
-            '& .MuiOutlinedInput-root .MuiOutlinedInput-notchedOutline': {
-                color: 'white',
-                borderColor: '#6678F6'
-            },
-            '&:hover .MuiOutlinedInput-input': {
-                color: 'white'
-            },
-            '& .MuiSelect-icon': {
-                color: '#6678F6'
-            }
+const styles = {
+    formControl: {
+        'margin': 1,
+        'minWidth': 220,
+        '& .MuiOutlinedInput-input': {
+            color: 'white'
+        },
+        '& .MuiInputLabel-root': {
```

Comment:
These are duplicated throughout the application. It would be little effort moving the styles for .MuiInputLabel-root  .MuiOutlinedInput-input.MuiOutlinedInput-root and all the rest  up to global theme level and only overridden in the few cases it needs. Quick global search will show you

Reply:
I've gone in and replaced all the textField instances to use the themedTextField component that actuall has these styles.